### ISSUE TITLE
Unbreakable doesn't work on sword

### ISSUE THREAD

**Issue by WarnDa** [EssentialsX/Essentials#3727]
### Information

**Full output of `/ess version`**:

```
[Tue, 13. Oct 2020 13:15:03 CEST INFO] Server version: 1.12.2-R0.1-SNAPSHOT git-Paper-1618 (MC: 1.12.2)
[Tue, 13. Oct 2020 13:15:03 CEST INFO] EssentialsX version: 2.18.1.23
[Tue, 13. Oct 2020 13:15:03 CEST INFO] PlaceholderAPI version: 2.10.9
[Tue, 13. Oct 2020 13:15:03 CEST INFO] LuckPerms version: 5.1.93
[Tue, 13. Oct 2020 13:15:03 CEST INFO] Vault version: 1.7.3-b131
[Tue, 13. Oct 2020 13:15:03 CEST INFO] EssentialsXSpawn version: 2.18.1.23```


**EssentialsX config**: https://gist.github.com/WarnDangerous/58d93aa1a25c86e7c82a6864281202c2

### Details

**Description**  
unbreakable flag doesn't work on diamond sword

**Steps to reproduce**  
I give myself a diamond_sword with "unbreakable" and I hit mobs (1300 mobs approximately, to go to "0 durability"

**Expected behavior**  
The tool shouldn't disapear

**Comment by pop4959** [comment]
Please provide more information, as you have not provided any logs or commands to help us figure out if this is a bug or not.

**Comment by WarnDa** [comment]
I just give myself a diamond sword : 
/egive username diamond_sword 1 unbreakable

It works, I receive an unbreakable sword, but when I use the 1561 durability, the item dissapear without reason.
I've tried and it ONLY happen to SWORD, not to other tools, and it only appear in minecraft 1.12.2, not in 1.15.

I can't provide logs as it won't help you, there is nothing in console/logs, just the item dissapear like if it wasn't unbreakable.

**Comment by pop4959** [comment]
Does this happen without any other plugins installed? This still doesn't sound like it would be a bug in Essentials.

**Comment by JRoy** [comment]
Closed in favor of #1105
